modules/compliance-operator-configure-scanning.adoc

```diff
@@ -58,6 +59,17 @@ $ scansettingbinding.compliance.openshift.io/cis-compliance created
 .. Click *+* to open the *Import YAML* page.
 .. Paste the YAML from the previous example and then click *Create*.
 
-.Additional resources
-* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/security_and_compliance/compliance-operator#understanding-compliance-operator[Understanding the Compliance Operator]
-* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/security_and_compliance/compliance-operator#compliance-operator-scans[Compliance Operator scans] in {ocp}
+.Verification
+
+. Run a compliance scan in {product-title-short}.
++
+For more information about running a compliance scan by using Compliance (1.0), see "Running a compliance scan" in the "Additional resources" section.
++
+[IMPORTANT]
```

Comment:
I think this should be moved to the top of the `Configuring the ScanSettingBinding object` section